inspector/src/components/sceneExplorer/entities/gui/guiTools.ts

```diff
@@ -0,0 +1,54 @@
+import { AdvancedDynamicTexture } from "babylonjs-gui/2D/advancedDynamicTexture";
+import { Engine } from "babylonjs/Engines/engine";
+import { Tools } from "babylonjs/Misc/tools";
+
+
+declare var GUIEDITOR: any;
+declare var BABYLON: any;
+
+export class GUITools {
+    private static guiEditor: any = null;
+    /** Get the inspector from bundle or global */
+    private static _getGlobalGUIEditor(): any {
+        // UMD Global name detection from Webpack Bundle UMD Name.
+        if (typeof GUIEDITOR !== 'undefined') {
+            return GUIEDITOR;
+        }
+
+        // In case of module let's check the global emitted from the editor entry point.
+        if (typeof BABYLON !== 'undefined' && typeof BABYLON.GUIEditor !== 'undefined') {
+            return BABYLON;
+        }
+
+        return undefined;
+    }
+
+    public static async EditAdvancedDynamicTexture(adt: AdvancedDynamicTexture) {
+        if (!this.guiEditor) {
+            if (typeof BABYLON !== 'undefined') {
+                // we are in UMD environment
+                this.guiEditor = this.guiEditor || this._getGlobalGUIEditor();
+                if (typeof this.guiEditor == 'undefined') {
+                    const editorUrl = `https://unpkg.com/babylonjs-gui-editor@${Engine.Version}/babylon.guiEditor.js`;;

```

Comment:
will be good to allow to override this url in case you want to host it yourself.